src/openms/source/FORMAT/FASTAFile.cpp

```diff
@@ -132,7 +132,7 @@ namespace OpenMS
     f.readStart(filename);
     while (f.readNext(p))
     {
-      data.push_back(p);
+      data.push_back(std::move(p));
```

Comment:
no difference IMHO, since we're moving already constructed objects (as opposed to forwarding c'tor arguments)